ADAL/src/telemetry/ADTelemetryAPIEvent.m

```diff
@@ -162,27 +162,26 @@ - (void)addAggregatedPropertiesToDictionary:(NSMutableDictionary *)eventToBeDisp
 {
     [super addAggregatedPropertiesToDictionary:eventToBeDispatched];
     
-    NSArray* properties = [self getProperties];
-    for (ADTelemetryProperty* property in properties)
+    NSDictionary* properties = [self getProperties];
+    for (NSString* name in properties)
```

Comment:
Also, this logic easily could be genericized to take in an array of properties as well....